core/tests/performance_tests/base.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Common utilities for performance test classes"""
+
+import unittest
+
+from core.tests.performance_framework import selenium_driver
+from core.tests.performance_framework import process_data
+
+# pylint: disable=invalid-name
+selenium_data_fetcher = selenium_driver.SeleniumPerformanceDataFetcher
```

Comment:
I'm not keen on this pattern. I think I prefer using the classnames and module names directly. I don't think we use this pattern much now (wasn't aware we did at all) and I don't think it's something we want to encourage. My concern is that it results in code that isn't easily referenceable (e.g. if I see ``page_session_metrics`` I can't just search for that in the codebase and see the class it's interacting with). Also, it results in different usage patterns since we never name classes in snake case.